client/src/App.jsx

```diff
@@ -16,24 +19,36 @@ class App extends Component {
     return (
       <div className="App">
         <Router>
+          <ToastContainer
```

Comment:
you don't need to react-tostify you can use antd.notifiaction

Reply:
remove tostify :+1: